docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md

```diff
@@ -286,7 +286,7 @@ under the License.
 | 1    | 100  | beijing   | 1.1  |
 | 2    | 200  | shanghai  | 1.2  |
 | 3    | 300  | guangzhou | 1.3  |
-| 4    | 400  | chongqing | 1.4  |
+| 4    | \N   | chongqing | 1.4  |
```

Comment:
Again, the filtered data is 400, which should not be changed to null.
You should change the 4 in the first column to null

Reply:
Yes, I found that I corrected the wrong place.